app/session.server.ts

```diff
@@ -35,7 +40,101 @@ const { getSession, commitSession, destroySession } =
     },
   });
 
-export { getSession, commitSession, destroySession };
+const fiveMinutesInSeconds = 60 * 5;
+const fiveMinutesInMilliseconds = fiveMinutesInSeconds * 1000;
+
+const twoWeeksInSeconds = 60 * 60 * 24 * 14;
+const twoWeeksInMilliseconds = twoWeeksInSeconds * 1000;
+
+// Session last 2 weeks by default. For testing purposes, we can add a
+// DEBUG_SESSION environment variable to make the sessions last 5 minutes, which
+// is the minimum allowed time for a session
+const COOKIE_EXPIRATION = {
+  seconds: process.env.DEBUG_SESSION ? fiveMinutesInSeconds : twoWeeksInSeconds,
+  milliseconds: process.env.DEBUG_SESSION
+    ? fiveMinutesInMilliseconds
+    : twoWeeksInMilliseconds,
```

Comment:
We need the time in seconds and milliseconds because we're dealing with 2 APIs:
- the `Set-Cookie` header takes a duration in seconds
- Firebase's `createSessionCookie()` takes a duration in milliseconds